content/news/21.9release.md

```diff
@@ -0,0 +1,199 @@
+
+FML will now load anything that is available on the classpath in development, including libraries.
+Minecraft, mods, and game libraries[^1] will be loaded in a transformable module layer, whereas other libraries will remain available through the normal classloader.
+As a result, library dependencies can now be added through standard Gradle means without needing to add them to a special classpath:
+```diff
+  dependencies {
+      // This is still required to add the library in your jar and at compile+run time.
+      jarJar(implementation("org.commonmark:commonmark:0.21.0"))
+-     // This can be removed since implementation dependencies are already made available at runtime:
+-     additionalRuntimeClasspath "org.commonmark:commonmark:0.21.0"
+}
+```
+
+Adding dependencies to `additionalRuntimeClasspath` has no effect anymore.
+Future versions of MDG will likely disallow using it entirely in Minecraft 1.21.9 and newer.
+
+Note that the latest versions of the Gradle plugins are **2.0.110** for MDG and **7.0.192** for NG. Remember to update your Gradle plugin before or after the migration to Minecraft 1.21.9.
+
+### Other FML usage
+Modders making use of other advanced FML features, such as language loaders or other sorts of FML plugin developers, will be affected to a larger extent.
+Don't hesitate to come and ask for support in [the NeoForge Discord server](https://discord.neoforged.net) if you need help ugrading.
+
+## Transfer rework
+A large rework of the Transfer APIs was merged in NeoForge 21.9.1-beta. You can either update directly to the latest NeoForge version, or target 21.9.0-beta first so that you can adapt to the transfer rework separately. 
+
+This change affects the `IItemHandler`, `IFluidHandler`, `IFluidHandlerItem` and `IEnergyStorage` interfaces, the corresponding capabilities, and all the supporting classes.
+See [the corresponding blog post](21.9-transfer-rework) for a detailed explanation, including porting recommendations.
+
+## Level rendering changes
+Vanilla separated level rendering into an _extraction phase_ of render state from the level,
+followed by a _render phase_ that receives the render state and submits things to render.
+
+Except for debug renderers, in 1.21.9 all level rendering is thus migrated to a render state.
+
+### Entities
+`EntityRenderer`s were already migrated to a render state in 1.21.6. They now submit to a `SubmitNodeCollector` instead of a `MultiBufferSource`. The actual rendering will be handled later for improved batching.
+
+### Block entity renderers
+`BlockEntityRenderer`s have been migrated to the render state system as well. They now have an additional type parameter -- the associated render state -- and the old `render` method is replaced by 3 new methods:
+- `createRenderState`: creates a new instance of the render state. The render state should be a mutable object and will be reused.
```

Comment:
The render state is no longer reused on entity renderers and BERs, a new instance is created for every entity or BE on every extraction